### ISSUE TITLE
[videolectures.net] download error: unable to download video data: HTTP Error 403: Forbidden

### ISSUE THREAD

**Issue by yassersouri** [rg3/youtube-dl#3175]
Salam

When I try to download videos from `videolectures.net` the following error occurs:

```
› youtube-dl --verbose http://videolectures.net/pcw05_zisserman_1voci/
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['--verbose', 'http://videolectures.net/pcw05_zisserman_1voci/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2014.06.26
[debug] Python version 2.7.5 - Darwin-13.2.0-x86_64-i386-64bit
[debug] Proxy map: {}
[videolectures.net] pcw05_zisserman_1voci: Downloading XML
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1020, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 996, in dl
    return fd.download(name, info)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/common.py", line 291, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/bin/youtube-dl/youtube_dl/downloader/http.py", line 58, in real_download
    data = self.ydl.urlopen(request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1225, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 403: Forbidden
```

This might be a problem related to videolectures.net admins changing things in their website.

**Comment by naglis** [comment]
Closed by 0d93378. This should be fixed in youtube-dl versions 2015.02.17.2 and newer. Refer to [our FAQ](https://yt-dl.org/update) if you need help updating. Thanks for the report.

**Comment by yassersouri** [comment]
Thanks.
